<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuracion</class>
 <widget class="QWidget" name="Configuracion">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana</string>
  </property>
  <widget class="QGroupBox" name="gbHabilitaciones">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>221</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Habilitaciones</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <widget class="QCheckBox" name="checkBoxClicConL">
      <property name="text">
       <string>Click con L en lugar del índice</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkSecuenciaGestos">
      <property name="text">
       <string>Secuencia de gestos</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBoxMargenes">
      <property name="text">
       <string>Dibujar márgenes</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="checkBoxCorreccion">
      <property name="text">
       <string>Corrección de imagen</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkInteraciones">
      <property name="text">
       <string>Interacciones</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBoxFPS">
      <property name="text">
       <string>Mostrar fps</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="checkBoxMostrarDragAndDrop">
      <property name="text">
       <string>Mostrar Drag and Drop etc</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="checkBoxLabelGesto">
      <property name="text">
       <string>Label gesto</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="checkBoxEsqueleto">
      <property name="text">
       <string>Dibujar esqueleto</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbActualizar">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>540</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar JSON con estos datos</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbMonitor">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>351</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Tamaño y posición del monitor</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="5" colspan="2">
     <widget class="QRadioButton" name="rbPosArrDer">
      <property name="text">
       <string>Arriba/Derecha</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QRadioButton" name="rbPosArrIzq">
      <property name="text">
       <string>Arriba/Izquierda</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="5" colspan="2">
     <widget class="QRadioButton" name="rbPosAbaDer">
      <property name="text">
       <string>Abajo/Derecha</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QRadioButton" name="rbPosAbaIzq">
      <property name="text">
       <string>Abajo/Izquierda</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QRadioButton" name="rbPosCentro">
      <property name="text">
       <string>Centro</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lAnchoAltoImagen">
        <property name="text">
         <string>Ancho y alto (400 x 300)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leAnchoImagen">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leAltoImagen">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gbAvanzado">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Avanzado</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="lLambda">
      <property name="text">
       <string>_lambda (0.15)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leLambda">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="la_x">
      <property name="text">
       <string>a_x (0.5)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lea_x">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lb_x">
      <property name="text">
       <string>b_x (0.8)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leb_x">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="la_y">
      <property name="text">
       <string>a_y (0.3)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lea_y">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lb_y">
      <property name="text">
       <string>b_y (0.7)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="leb_y">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lAlphaCamara">
      <property name="text">
       <string>Alpha cámara (210)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="leAlphaCamara">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lAlphaTeclado">
      <property name="text">
       <string>Alpha teclado (0.85)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="leAlphaTeclado">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gbModo">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Modo Navegación / Modo Dibujo</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QRadioButton" name="rbNavegacion">
      <property name="text">
       <string>Navegación</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="rbDibujo">
      <property name="text">
       <string>Dibujo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gbDepuradores">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>560</width>
     <height>410</height>
    </rect>
   </property>
   <property name="title">
    <string>Depuradores</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="1">
     <widget class="QLineEdit" name="leTiempoParaAccionar">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="leMouseFactorCercano">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lMouseFactorCercano">
      <property name="text">
       <string>mouse_factor_cercano (Default: 30)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leMouseKeypoint">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lClicDerechoCercanoGesto">
      <property name="text">
       <string>clic_derecho_cercano_gesto (Default: 3)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="leClicKeypointZ">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leMouseGesto">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lClicCercanoGesto">
      <property name="text">
       <string>clic_cercano_gesto (Default: 7)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="leClicKeypointL">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lTiempoParaAccionar">
      <property name="text">
       <string>tiempo_para_accionar (para isListo_para_accionar) (Default: 20)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lMouseKeypoint">
      <property name="text">
       <string>control_mouse_keypoint (Default: 9)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="leClicCercanoGesto">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="lClicKeypointL">
      <property name="text">
       <string>clic_keypoint_L_antes_de_clic (Default: 9)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lClicKeypointZ">
      <property name="text">
       <string>clic_keypoint_z (Default: 8)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="leClicDerechoCercanoGesto">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lMouseGesto">
      <property name="text">
       <string>control_mouse_gesto (Default: 5)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="leVentanaClic">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="leVentanaClicL1">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="lVentanaClic">
      <property name="text">
       <string>ventana_clic (Default: 500)</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="lVentanaClicL1">
      <property name="text">
       <string>ventana_clic_L1 (Default: 1200)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
